cookies/templates/resources.html

```diff
@@ -52,21 +52,37 @@ <h4 class="panel-title">
     </div>
 </div>
 
-<div class="collection_resources">
-    <div class="pagination container text-center">
-        {% paginate %}
+<div class = "row">
+    <div class = "col-sm-1">
```

Comment:
Good idea to use the ``sm`` breakpoint. But you can use more space than that -- try ``col-sm-3`` or ``col-sm-4``